src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -67,15 +67,19 @@ public boolean isExit() {
     }
 
     public boolean isClientSpecific() {
-        return specific.equals(CommandSpecific.CLIENT);
+        return specific != null && specific.equals(CommandSpecific.CLIENT);
```

Comment:
I was thinking is instead of `specific != null` could we use `requireNonNull(specific)` instead. Just to make the code look cleaner, coz I feel that `specific != null` seems a bit arbitrary.

Reply:
Same for line 74, 78 and 82

Reply:
hmm this is because specific can allow null values for commands that arent specific such as help, exit, and potentially others that we implement down the road eventually

Reply:
i think to keep it cleaner it might be possible to implement another enum type perhaps something like `Others`. Thoughts on this?

Reply:
Then the default value can just be that

Reply:
Yea, would prefer `Others`. Dont think we should give any meaning to the `null` value